### DISCUSSION TITLE
The background image is not displaying when i publish me site using github.

### DISCUSSION THREAD

**Discussion by aayushibansal2001** [community/community#23579] [Other Feature Feedback, Questions, & Ideas]
<p>Hello Everyone!</p>
<p>I am facing problem in the site published using github. The background image is not getting displayed when i publish the site. However, it is displaying in my local computer. </p>
<p>The backround image is named as ‘4’ and is present in images folder. </p>
<p>I am not able to get where is the problem occuring and how to resolve it. </p>
<p>The link to my published site is: <a href="https://aayushibansal2001.github.io/Blog-Page/" rel="nofollow noopener">https://aayushibansal2001.github.io/Blog-Page/</a></p>
<p>The link to my github repository is:<a href="https://github.com/aayushibansal2001/Blog-Page" rel="nofollow noopener">https://github.com/aayushibansal2001/Blog-Page</a></p>

**Comment by r00t-44** [comment, marked as answer]
<p>Change your file format of image <strong>4.JPG to 4.jpg</strong> in your images folder. This error is occuring because you have used 4.jpg in your css files…</p>

**Reply by GiuseppeAderemi** [reply]
https://github.com/GiuseppeAderemi/Td-Project4-Css-Basics

The link to the github repo....The image is not showing

**Comment by aayushibansal2001** [comment]
<p>How do we convert from .JPG to .jpg? Aren’t both the same? </p>
<p>Also i have tried doing 4.JPG in my css file, but then also the problem didn’t resolve.</p>

**Comment by r00t-44** [comment]
<p>I think it is solved now…</p>
<p>And yes both .JPG and .jpg are same extensions but in some OS it became case sensitive and hence always preferable to use lower case extensions…</p>

**Comment by aayushibansal2001** [comment]
<p>Yes. The problem is resolved now. Thanks for the help.</p>

**Comment by valentina-caponi** [comment]
<p>Hello, I am facing more or less a similar problem, because my profile picture is not displaying when I publish my website and it shows like this:<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/b9180ea191a36a19d609eb134b6f17e601377a92.png" data-download-href="/uploads/short-url/qppScB8Qa97hoyx2HEB2BEMODlM.png?dl=1" title="Screenshot 2020-07-06 at 11.19.19" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/67897366/181078011-ee72b38e-1c7e-4814-948d-2a575332805c.png" alt="Screenshot 2020-07-06 at 11.19.19" data-base62-sha1="qppScB8Qa97hoyx2HEB2BEMODlM" width="674" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/b9180ea191a36a19d609eb134b6f17e601377a92_2_674x500.png, https://aws1.discourse-cdn.com/github/original/2X/b/b9180ea191a36a19d609eb134b6f17e601377a92.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/b/b9180ea191a36a19d609eb134b6f17e601377a92.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/b9180ea191a36a19d609eb134b6f17e601377a92_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screenshot 2020-07-06 at 11.19.19</span><span class="informations">718×532 2.27 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I have saved it as “profile-picture.png” in the images folder, so why does it not work? Please help me!</p>
<p>Link to website: <a href="https://valentina-caponi.github.io/my-profile/" rel="nofollow noopener">https://valentina-caponi.github.io/my-profile/</a></p>
<p>Thank you!!</p>

**Comment by aayushibansal2001** [comment]
<p>you should write src=“profile-picture.png” instead of src=“images/vals.jpeg”</p>

**Comment by valentina-caponi** [comment]
<p>Hi, thanks for answering! I have done that. I have written src=“images/profile-picture.png” on HTML, but the image still does not show.</p>
<p>And do you know why when I have to load the images folder on Github it does not accept the folder but only the image?</p>

**Comment by aayushibansal2001** [comment]
<p>Try writing src=“profile-picture.png” and not src=“images/profile-picture.png” because you don’t have images folder uploaded.</p>
<p>And also the folders are not accepted. so to make the folder you can click on the create file button and and the name of the folder and press enter. Then add any file and commit that. After that you can add the images in that folder created.</p>
<p>I hope it helps.</p>

**Comment by valentina-caponi** [comment]
<p>Thank you a lot! It actually worked.</p>
<p>However, I don’t understand why it didn’t accept my images folder. I had created it and added the image inside as you said. Not sure.</p>

**Comment by den864** [comment]
<p>I am facing same issue. In my case .jpg and .JPG are not the problem though in github pages background image is not showing.</p>
<p>here is my code:</p>
<p>.showcase-area{<br>
height: 50vh;<br>
background: url("/images/header-image2.jpg");<br>
background-position: center;<br>
background-size: cover;<br>
background-repeat: no-repeat;<br>
}</p>
<p>Link: <a href="https://den864.github.io/restaurant-web/" rel="noopener nofollow ugc">https://den864.github.io/restaurant-web/</a><br>
repo : <a href="https://github.com/den864/restaurant-web" rel="noopener nofollow ugc">https://github.com/den864/restaurant-web</a></p>
<p>thank you</p>

**Comment by otter-computer** [comment]
<p>It looks like a relative path issue. The browser is looking inside the <code>css</code> folder for the <code>images</code> directory instead of the root of your site. You’ll need to move up a directory to the root using a relative link before it’ll work:</p>
<pre><code class="lang-auto">background: url("../images/header-image2.jpg");
</code></pre>

**Reply by welangaieric** [reply]
worked

**Comment by den864** [comment]
<p>Yeah it’s working. I almost forget that btw thanks :slight_smile:</p>

**Comment by excela01** [comment]
<p>I have similar problems. I changed the file from .jpG to .jpG in my GitHub repo. Still no changes reflect… still blank…</p>
<p>Link to my pages: <a href="http://excela01.github.io/myFamilyInfoBasic/" class="inline-onebox" rel="noopener nofollow ugc">FAMILY INFO</a></p>
<p>Link to the repository :</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://GitHub.com/excela01/myFamilyInfoBasic/">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://GitHub.com/excela01/myFamilyInfoBasic/" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/81013396/181078019-23051853-1a40-4df7-a226-6e8630e7b929.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/765209f8d6633bf0971deb064d8c4ebba0b504cf_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/7/6/765209f8d6633bf0971deb064d8c4ebba0b504cf_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/7/6/765209f8d6633bf0971deb064d8c4ebba0b504cf.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/7/6/765209f8d6633bf0971deb064d8c4ebba0b504cf_2_10x10.png"></div>

<h3><a href="https://GitHub.com/excela01/myFamilyInfoBasic/" target="_blank" rel="noopener nofollow ugc">GitHub - excela01/myFamilyInfoBasic: just a little family game to entertain...</a></h3>

  <p>just a little family game to entertain members of my family, by putting in their names and gettting certain outputs - GitHub - excela01/myFamilyInfoBasic: just a little family game to entertain mem...</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by Robmel12** [comment]
<p>I’m also having this problem. I’ve tried a number of different paths but I noticed in chrome developer tools under source the only image file that is loading onto the site it my logo.png file. All pictures with jpg formatting won’t show.<br>
I’ve tried dist/images/teacher.jpg …/dist/…, ./dist/…, …/images/…, etc. nothing works. It doesn’t matter cause the file says the pictures aren’t even there.</p>
<p><a href="https://robmel12.github.io/Inter-Act_Website/" class="inline-onebox" rel="noopener nofollow ugc">Inter-Act Home-page</a></p>

**Comment by Robmel12** [comment]
<p>nvm solved it. I thought I tried …/images/teacher.jpg but i guess not. Working now!</p>

**Comment by Anjal-Rajeev** [comment]
Hi ,  
https://github.com/Anjal-Rajeev/To-Do-list  . this is my repository link 
https://anjal-rajeev.github.io/To-Do-list/index.html   . this is the hosted link. My problem is , background image is not showing in index.html and a CSS file named todo.css is also not working. Can anyone help me to solve this.

**Reply by Chiiiemela** [reply]
Me too my background image isn't showing up 
Can anyone help me?

**Reply by welangaieric** [reply]
Try (../imagename)

**Comment by santoshjobs2023** [comment]
The background image perfectly shows in localhost but doesn't show in Github ...

my github website- https://santoshjobs2023.github.io/myportfolio/

my repositry - https://github.com/santoshjobs2023/myportfolio

plase help me here
